server/block/src/chest.rs

```diff
@@ -223,12 +219,8 @@ fn load_inventory(slots: &[InventorySlot]) -> Inventory {
     let inv = Inventory::chest();
 
     for slot in slots {
-        if let Some(item) = Item::from_identifier(&slot.item) {
-            if let Err(e) = inv.set_item_at(
-                Area::Chest,
-                slot.slot as usize,
-                ItemStack::new(item, slot.count as u8),
-            ) {
+        if let Some(_item_identifier) = Item::from_identifier(&slot.item) {
```

Comment:
This could be changed to `if Item::from_identifier(&slot.item).is_some()` for clarity.